corehq/apps/smsforms/models.py

```diff
@@ -102,7 +102,7 @@ class SQLXFormsSession(models.Model):
     Keeps information about an SMS XForm session.
     """
     # generic properties
-    couch_id = models.CharField(null=True, blank=True, db_index=True, max_length=50)
+    couch_id = models.CharField(db_index=True, max_length=50)
```

Comment:
I realized that because of stuff like this: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/sms/models.py#L53 it'd be easier to just keep populating this ID than to remove it once couch models are gone, so disallowed nulls and blanks. i confirmed on prod that there are none